com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md

```diff
@@ -24,7 +24,7 @@ To create a new Layered Lit Material, navigate to your Project's Asset window, r
 | **Double-Sided**          | Enable the checkbox to make HDRP render both faces of the polygons in your geometry. For more information about the feature and for the list of properties this feature exposes, see the [Double-Sided documentation](Double-Sided.html). |
 | **Alpha Clipping**        | Enable the checkbox to make this Material act like a [Cutout Shader](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterRenderingMode.html). Enabling this feature exposes more properties. For more information about the feature and for the  list of properties this feature exposes, see the [Alpha Clipping documentation](Alpha-Clipping.html). |
 | **Material Type**         | Allows you to give your Material a type, which allows you to customize it with different settings depending on the **Material Type** you select. For Layered Lit Materials, you can only use the **Subsurface Scattering**, **Standard**, or **Translucent** **Material Type**. For more information about the feature and for the list of properties each **Material Type** exposes, see the [Material Type documentation](Material-Type.html). |
-| **Receive Decals**        | Enable the checkbox to allow HDRP to draw decals on this Material’s surface. |
+| **Decal Layer Mask**      | Decal Projectors with matching Decal Layer Mask will affect this material (See [Decal Projector](Decal-Projector.html)). Decal meshes will affect this material if the value is different from `None`. |
```

Comment:
The layer mask that specifies which Decals can affect this Material. Decal Projectors that have a matching Decal Layer Mask affect this Material. Decal Meshes affect this Material if the value is different from `None`.

Can you apply this to all the other Decal Layer Mask entries in this PR?

Reply:
>Can you apply this to all the other Decal Layer Mask entries in this PR?
sure, thanks